worker/transfer.go

```diff
@@ -96,17 +115,34 @@ func parallelUploadSlab(ctx context.Context, shards [][]byte, hosts []sectorStor
 		}
 	}
 	if rem > 0 {
-		return nil, errs
+		return nil, nil, errs
+	}
+
+	// make hosts map
+	hostsmap := make(map[types.PublicKey]int)
```

Comment:
hostsMap

Reply:
hm kind of got in the habit of not uppercasing `map`, it's all over the codebase, I'll update all occurrences